_PlatformOS/getting-started/pages/adding-visuals-pages-javascript.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Adding Visuals to Pages: Javascript
+permalink: /get-started/pages/adding-visuals-pages-javascript
+---
+This guide will help you add visuals (images) to your pages using Javascript. 
+
+## Requirements
+To follow the steps in this tutorial, you should be familiar with the required directory structure for your code base, and understand the concepts of pages, layouts, and assets. The example in this tutorial modifies the layout `application.liquid` used in the previous tutorial. 
+
+* [Required Directory Structure]()
+* [Pages]()
+* [Layouts]()
+* [Assets]()
+* [Adding Visuals to Pages: Liquid]() (previous tutorial) 
+
+## Steps 
+
+Adding visuals to your pages using Javascript is a three-step process:
+
+1. Prepare an image
+2. Add Javascript to your layout 
+3. Check page source
+
+### Step 1: Prepare an image
+Put an image into the `marketplace_builder/assets/images` directory, e.g. logo.svg. 
+
+### Step 2: Add Javascript to your layout
+
+Add a piece of script that will console log the image URL. This would normally be your plugin loaded at the end of the page, or on the jQuery ready event after the HTML has loaded. 
+
+You can pass the URL of the asset (image) using the `data` attribute and the `{{ asset_url }}` filter, that takes a string as an argument, and returns a string representing the URL to this asset served by our CDN.  
```

Comment:
We shouldnt describe filter as `{{ filter }}` because it would throw an error.

{{ x }} means variable - in this case {{ asset_url }} is a variable with hash inside ( https://github.com/mdyd-dev/nearme-documentation/pull/5#discussion_r185488869 )

Reply:
Thanks! I hope I got it right, please check my edits.